package.json

```diff
@@ -135,6 +136,7 @@
     "http-node": "^1.2.0",
     "prettier-bytes": "^1.0.4",
     "qr-image": "^3.2.0",
+    "query-string": "^6.1.0",
```

Comment:
cc @diracdeltas for security review

Reply:
is it not possible here to use node's builtin querystring module? https://nodejs.org/api/querystring.html

Reply:
Sorry, my bad, I'll revert https://github.com/brave/brave-core/pull/405/commits/86fb70be5473b9417367828f246c5180f9885397 and use this one instead.